lib/dotcom_web/components/trip_planner/itinerary_summary.ex

```diff
@@ -54,12 +60,16 @@ defmodule DotcomWeb.Components.TripPlanner.ItinerarySummary do
   # No routes: this is a walking leg
   defp leg_icon(%{routes: [], walk_minutes: _} = assigns) do
     ~H"""
-    <span class={[
-      "flex items-center gap-1 text-sm font-semibold leading-none whitespace-nowrap py-1 px-2 rounded-full border-[1px] border-gray-light",
-      @class
-    ]}>
-      <.icon name="person-walking" class="h-4 w-4" />
-      <span>{@walk_minutes} min</span>
+    <span
+      aria-label={"#{@walk_minutes} minutes walk"}
```

Comment:
📓 Walking

Reply:
I _think_ this would be more natural as `minute` rather than `minutes`. "I'm going to go on a five minutes walk" doesn't sound right to me.